data/authenticjobs.js

```diff
@@ -1,5 +1,10 @@
 const axios = require('axios'); 
-const query_commands = require('./database/database.js')
+const query_commands = require('./database/database.js'); 
+let dateoption = {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric'
+  }; 
 
 const ROOT_URL = 'https://authenticjobs.com/api/?api_key='; 
 const KEY = 'c1f69b196fc568a7850bbb092eef2748'; 
```

Comment:
Long-term, it might be wise to avoid hard-coding api keys into the code, in case they eventually get revoked or overused. One way to do it: include instructions on creating a `.env` file, and add `.env` to your .gitignore. Your Readme can include your example api keys to make it easy for devs to use it.  See [dotenv](https://www.npmjs.com/package/dotenv).